examples/src/main/java/com/basistech/rosette/examples/EntitiesExample.java

```diff
@@ -35,16 +36,19 @@ public static void main(String[] args) {
 
     private void run() throws IOException, RosetteAPIException {
         String entitiesTextData = "Bill Murray will appear in new Ghostbusters film: Dr. Peter Venkman was spotted filming a cameo in Boston this… http://dlvr.it/BnsFfS";
-
-        RosetteAPI rosetteApi = new RosetteAPI(getApiKeyFromSystemProperty(), getAltUrlFromSystemProperty());
-        rosetteApi.setOptions(null);
-        rosetteApi.setGenre("social-media");
+        RosetteAPI rosetteApi = new RosetteAPI.Builder()
+                                .apiKey(getApiKeyFromSystemProperty())
+                                .alternateUrl(getAltUrlFromSystemProperty())
```

Comment:
If api key is null, a rosette exception is thrown saying authentication is needed, I've added a check for a null alternateUrl